packages/lego-bricks/src/components/Tabs/TabContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import cx from 'classnames';
+import { Flex } from '../Layout';
+import styles from './TabContainer.module.css';
+import type { ReactNode } from 'react';
+
+type Props = {
+  className?: string;
+  lineColor?: string;
+  children?: ReactNode;
+};
+
+export const TabContainer = ({ className, lineColor, children }: Props) => (
+  <Flex wrap className={cx(styles.container, className)}>
+    {children}
+    <div className={styles.spacer} style={{ borderColor: lineColor }} />
```

Comment:
Feels a bit off to see this everywhere on the webapp as we previously have not used it a lot. Anyways, I'm not sure if I dislike it or if it's just new - so I'm not opposed to keep it to see how it feels.

Reply:
Idk, I like it. Some of the pages look a bit weird with it though, especially the profile-page, so I'm not opposed to making it optional either.